### ISSUE TITLE
Charge detection issue

### ISSUE THREAD

**Issue by pmirabel** [jpraus/floower#12]
Hello! 

Just upgraded my Floower. Thank you again for this huge upgrade! I can't wait to play with WiFi :) but ... There seems to be an issue (or something I didn't understood well).  

- I am using an "old" (few years) 12W mac charger with my floower.     
- BT connection works fine and I can see the "battery charging notification" on my iPhone Floower app homepage.  
- My issue is that settings page complains "_Please charge your floower to enable wifi_" 
  - as long as a red warning logo just next to the above mentionned charging animation).  
- On the floower itself, i cannot see no red led shining.  

best regards :) 

NB: My floower is 24/24 connected to power, used (very) approx.  6 hours a week and and is a rev6.

**Comment by jpraus** [comment]
Hi! That is a pretty old lady! I love that you still have it and use it :) I have Floower rev 6 as well so will test the latest upgrade for compatibility, thank you for reporting!

**Comment by pmirabel** [comment]
> That is a pretty old lady! I love that you still have it and use it

Yeah, unfortunately, you heard from me only when there are issues with my floower, but we are very happy with it. I have to admit though that I use it "offline, just using the leaf sensor to toggle light and opening" by now, because of the lack of wifi connection  :)   
 
I cannot wait to connect it to internet to enable some fun "notifications" with it.

If there is an hardware issue with my floower I would suggest to add an "override power security switch" with proper warnings in the app :) (or not, if it is just for me ...)

See you soon.
